.github/settings.yml

```diff
@@ -0,0 +1,100 @@
+# https://github.com/apps/settings
+
+repository:
```

Comment:
What about the name?

```suggestion
repository:
  name: docs
```

Reply:
I think I am going to keep it as generic as possible to try to have the same settings for all repositories

Reply:
and what about the `description` and `homepage`?

Reply:
Yes, I am going to remove them too